javascript-forms/form-events.js

```diff
@@ -0,0 +1,43 @@
+function handleFocus(event) {
+  console.log('focus event fired');
+  console.log(event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blur event fired');
+  console.log(event.target.name);
+}
+
+function handleInput(event) {
+  console.log('input event fired');
+  console.log(event.target.name);
+}
+
+function handleSubmit(event) {
+  event.preventDefault();
+  var formData = {};
+  formData.name = $form.elements.name.value;
+  formData.email = $form.elements.email.value;
+  formData.message = $form.elements.message.value;
+  console.log('formData', formData);
+  $form.reset();
+}
```

Comment:
This portion should be being done in the `form-values.js` file, along with the `submit` event listener.